src/rust/engine/process_executor/src/main.rs

```diff
@@ -372,9 +372,13 @@ async fn main() {
   .expect("Error executing");
 
   if let Some(output) = args.materialize_output_to {
+    // NB: We use `destination` as the root directory, because there is no need to
+    // memoize a check for whether some other parent directory is hardlinkable.
```

Comment:
I'm a bit confused by why this decision is notable. What other option might there be other than `output`? Could you expand?

Reply:
Just that in most use cases, the directory we're materializing to is a child of some other configured directory... but not in the case of a one-shot CLI tool like `process_executor`.

Reply:
Ah, okay. Thanks